dali/pipeline/operators/fused/crop_mirror_normalize.h

```diff
@@ -51,7 +49,7 @@ class CropMirrorNormalize : public Operator<Backend> {
     if (!has_mirror_) {
       mirror_.Resize({batch_size_});
```

Comment:
Maybe we can obtain spec.GetArgument<int>("mirror") once before for loop and avoid calling it again for every iteration as value it the same.

Reply:
That is not true, this argument may be a tensor and so its value may change per sample.

Reply:
So spec.GetArgument<int>("mirror") is iterating over elements?

Reply:
No, but `spec.GetArgument<int>("mirror", spec, i)` would give you `i`-th element.

Here I basically fix a bug where if you do not specify tensor argument for mirror it would always not perform mirror instead of taking the value specified by the user.

Reply:
We do populate mirror_ with actual data in `SetupSharedSampleParams`.